app/actions/remote/team.ts

```diff
@@ -475,3 +475,12 @@ export async function getTeamMembersByIds(serverUrl: string, teamId: string, use
         return {error};
     }
 }
+
+export const buildTeamIconUrl = (serverUrl: string, teamId: string, timestamp = 0) => {
+    try {
+        const client = NetworkManager.getClient(serverUrl);
```

Comment:
For my own curiosity, is it the call to `getClient` that can error here? I'm not familiar with how the `NetworkManager` works

Reply:
Yes. Getting the client may throw if there is (or we cannot create) a client for that server url.